dist/scripts/api-console.js

```diff
@@ -756,12 +756,20 @@
           $scope.listArray = $scope.list;
         }
 
+        var getArrayTypes = function(arrayType) {
+          if (arrayType.items.type || Array.isArray(arrayType.items.type)) {
```

Comment:
It should be this way, the field items represents the type of the array, so it should always be defined. Furthermore, the previous code also assumed that it will always be defined.